PeakCoreData/Core/Protocols/ManagedObjectType.swift

```diff
@@ -158,16 +158,14 @@ public extension ManagedObjectType {
 
 public extension ManagedObjectType where Self: UniqueIdentifiable {
     
-    typealias UniqueKeyValueType = Any
-    
     /**
      - Note: The managed object must conform to UniqueIdentifiable.
```

Comment:
This comment is wrong now.

Reply:
Is it? The managed object does have to conform to `UniqueIdentifiable`. It might be kind of redundant as we're in an extension that enforces `UniqueIdentifiable`

Reply:
Oh sorry, I think I've conflated UniqueKeyValueType and UniqueIdentifiable.